src/test/java/seedu/address/model/order/DeliveryDateTimeTest.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.model.order;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class DeliveryDateTimeTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new DeliveryDateTime(null));
+    }
+
+    @Test
+    public void constructor_invalidDeliveryDateTime_throwsIllegalArgumentException() {
+        String invalidDeliveryDateTime = "15/12/2022 1432"; //Invalid date time format
```

Comment:
Suggest putting this in CommandTestUtil to reference instead. Would be able to improve SLAP?